src/CheckoutSdk/CardIssuing/CardHolders/CardHolderCardsResponse.cs

```diff
@@ -0,0 +1,11 @@
+using Checkout.CardIssuing.Cards;
+using Checkout.Common;
+using System.Collections.Generic;
+
+namespace Checkout.CardIssuing.CardHolders
+{
+    public class CardHolderCardsResponse : Resource
```

Comment:
Cardholder is one word. There is no need for the CamelCase there (should be CardholderCardsResponse). Please correct this in all the other classes